src/parser/parse_tree.h

```diff
@@ -3017,6 +3017,7 @@ struct pt_killstmt_info
 struct pt_sort_spec_info
 {
   PT_NODE *expr;		/* PT_EXPR, PT_VALUE, PT_NAME */
+  PT_NODE *prev_expr;		/* PT_EXPR, PT_VALUE, PT_NAME */
```

Comment:
In pt_init_sort_spec, prev_expr initialization is required.

Reply:
I checked that memset is being called in parser_init_node.

Reply:
It is already initialized in parser_init_node().